img/v/vicroads.vic.gov.au.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 675.71 675.71"><path fill="#3cb54a" d="M0 0s371.23 370.69 556.53 555.81c-3.74 4.77-24.11 29.79-59.88 33.55-27.88 2.93-48.63-8.81-55.22-12.91Q220.78 355.82.14 135.16H0v540.55h675.71V0Zm397.56 147h95.48a81.11 81.11 0 0 1 64.52 70.7l.48 102.39a97.91 97.91 0 0 0-25.26-50.7c-12.63-13.37-32.84-22.25-47.71-26.32-2.49-.69-5.24-1.31-7.35-1.73a49.84 49.84 0 0 0-8.59-.8c-59.87-.34-104.09.8-164 .46Zm65 296.77q.51-94.46 1-188.9c13.44 1 41.83 5.35 61.42 27.35 11.48 12.89 17.39 26.94 23.32 46.78 6.48 21.65 7 40.65 7.65 53.87l2.58 152.25Z"/></svg>
```

Comment:
Here's a version as seen on their website (with gradients):
![vicroads vic gov au](https://user-images.githubusercontent.com/520841/170862638-9a489751-6e2b-4ed7-8926-0aae4f1493d3.svg)

```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 121.71 102.8"><defs><linearGradient id="f" x1="0" x2="1" y1="0" y2="0" gradientTransform="scale(-13.85601 13.85601) rotate(20 -185.08 -67.6)" gradientUnits="userSpaceOnUse" spreadMethod="pad"><stop offset="0" stop-color="#8cc63f"/><stop offset=".12" stop-color="#8cc63f"/><stop offset=".38" stop-color="#54b948"/><stop offset=".66" stop-color="#00a651"/><stop offset=".95" stop-color="#005826"/><stop offset="1" stop-color="#005826"/></linearGradient><linearGradient id="d" x1="0" x2="1" y1="0" y2="0" gradientTransform="scale(40.41433 -40.41433) rotate(15.04 82.57 30.76)" gradientUnits="userSpaceOnUse" spreadMethod="pad"><stop offset="0" stop-color="#8cc63f"/><stop offset=".13" stop-color="#8cc63f"/><stop offset=".32" stop-color="#54b948"/><stop offset=".6" stop-color="#00a651"/><stop offset=".86" stop-color="#005826"/><stop offset="1" stop-color="#005826"/></linearGradient><linearGradient id="b" x1="0" x2="1" y1="0" y2="0" gradientTransform="scale(12.71464 -12.71464) rotate(-60 -36.62 -63.68)" gradientUnits="userSpaceOnUse" spreadMethod="pad"><stop offset="0" stop-color="#f0f0f2"/><stop offset=".03" stop-color="#f0f0f2"/><stop offset=".9" stop-color="#243444"/><stop offset="1" stop-color="#243444"/></linearGradient><clipPath id="e" clipPathUnits="userSpaceOnUse"><path d="m465.56 826.54-4.97-4.96h6.88l1.43-.01c.67-.02 1.3-.09 1.95-.3.86-.3 1.62-.84 2.14-1.58.45-.63.73-1.38.88-2.13v4.46c0 2.26-1.42 4.52-4.26 4.52z"/></clipPath><clipPath id="c" clipPathUnits="userSpaceOnUse"><path d="M439.37 832.32s26.19-26.08 28.15-28.05c2.8-2.8 6.34.68 6.34.68l-27.43 27.37z"/></clipPath><clipPath id="a" clipPathUnits="userSpaceOnUse"><path d="m468.9 809.9 4.97-4.96v12.61c-.15.76-.42 1.5-.87 2.14a4.32 4.32 0 0 1-2.15 1.57c-.64.22-1.28.29-1.95.31V809.9"/></clipPath></defs><g clip-path="url(#a)" transform="matrix(3.52778 0 0 -3.52778 -1550 2936.25)"><path fill="url(#b)" d="m468.9 809.9 4.97-4.96v12.61c-.15.76-.42 1.5-.87 2.14a4.32 4.32 0 0 1-2.15 1.57c-.64.22-1.28.29-1.95.31V809.9"/></g><g clip-path="url(#c)" transform="matrix(3.52778 0 0 -3.52778 -1550 2936.25)"><path fill="url(#d)" d="M439.37 832.32s26.19-26.08 28.15-28.05c2.8-2.8 6.34.68 6.34.68l-27.43 27.37z"/></g><g clip-path="url(#e)" transform="matrix(3.52778 0 0 -3.52778 -1550 2936.25)"><path fill="url(#f)" d="m465.56 826.54-4.97-4.96h6.88l1.43-.01c.67-.02 1.3-.09 1.95-.3.86-.3 1.62-.84 2.14-1.58.45-.63.73-1.38.88-2.13v4.46c0 2.26-1.42 4.52-4.26 4.52z"/></g></svg>
```

Reply:
Good find, I sent ages looking for that. Ended up making a trashy version of one they sometimes use